include/pokeagb/overworld/item.h

```diff
@@ -449,6 +449,18 @@ POKEAGB_EXTERN u16 get_item_quantity(enum Item item);
  */
 POKEAGB_EXTERN bool check_item(enum Item item, u8 quantity);
 
+/**
+ * Adds a item to the player's bag
+ * @address{BPRE,0809A084}
+ */
+POKEAGB_EXTERN void bag_add_item(u16 item_id, u16 unknown_maybe_quantity_or_pocket);
```

Comment:
This is not `void`:

The ASM sequence
```asm
ROM:0809A1D2                 POP     {R1}
ROM:0809A1D4                 BX      R1
```
indicates that a function has a return value since it is not clobbering `r0`.